src/pybind/mgr/cephadm/nfs.py

```diff
@@ -107,20 +111,43 @@ def create_rados_config_obj(self, clobber=False):
     def get_ganesha_conf(self):
         # type: () -> str
         return '''# generated by cephadm
+NFS_CORE_PARAM {{
+        Enable_NLM = false;
+        Enable_RQUOTA = false;
+        Protocols = 4;
```

Comment:
is this something that cephadm should control, or @varshar16 ?

Reply:
It's required for nfs grace in the the bootstrap config ... we need to investigate options for allowing this to be overwritten by consumers of the orchestrator interface (rook/rook#5122)